aten/src/ATen/cpu/vec/vec512/intrinsics.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#if (defined(__clang__) || defined(__GNUC__)) && (defined(__x86_64__) || defined(__i386__))
+/* GCC or Clang, targeting x86/x86-64 */
+#include <x86intrin.h>
+#elif defined(_MSC_VER)
+/* Microsoft C/C++-compatible compiler */
+#include <intrin.h>

```

Comment:
Why can't we just use the vec256/intrinsics.h header?

Reply:
We can. The vec256 header has some macros that we don't use in the AVX512 code.
But I guess a unified header would indeed make more sense.